### ISSUE TITLE
Allow user to really start task from new task menu

### ISSUE THREAD

**Issue by badb** [golemfactory/clay#335]
So clicking "start task" from tasks menu won't be necessary. 
There are some ux problems, it takes too long. We should probably put GUI in another thread before we can support this.

**Comment by magdasta** [comment]
Is this one still relevant? Or changing the <kbd>Start task</kbd> to <kbd>Create task</kbd> in new task menu resolved it?

**Comment by badb** [comment]
Let's close it. When we get more feedback about interface / plan the new one we will think again about this.
